Chart: update Overview (#7056)
DevExpress/devextreme-documentation: concepts/05 UI Components/Chart/35 Legend/00 Overview.md

---
title: Chart - Legend Overview
---

> The legend is a chart component that helps a user identify series. It contains several colored items, one per series. 

Reviewer: ```suggestion
The legend helps users identify series by displaying a colored item for each series.
```
